addons/web/static/src/views/calendar/filter_panel/calendar_filter_panel.js

```diff
@@ -68,7 +68,7 @@ class FilterAutoComplete extends Component {
             context: {},
             domain: this.props.domain,
             onSelected: (...args) => {
```

Comment:
I would name the first parameter like this. Otherwise, the fix is ok :+1: 

```suggestion
            onSelected: (id) => {
                this.onSelect({value: id[0]});
```